core/src/test/java/org/elasticsearch/search/aggregations/metrics/scripted/InternalScriptedMetricTests.java

```diff
@@ -19,7 +19,10 @@
 
 package org.elasticsearch.search.aggregations.metrics.scripted;
 
+import com.google.common.base.Supplier;
```

Comment:
No, good catch. Seems to do the same thing as java.util.function.Supplier apparently though.